eo2js/test/commands/transpile.test.js

```diff
@@ -57,17 +64,40 @@ describe('transpile', function() {
     }
     it('should create transpiled XMIRs', function() {
       const traspiled = '8-transpile/com/eo2js/simple.xmir'
+      prepare()
```

Comment:
@HaidarJbeily7 you can replace usage of `prepare` method with default `beforeEach` provided by `mochajs`